src/main/g8/README.md

```diff
@@ -46,13 +46,13 @@ to play with small snippets of code:
 > console
 [info] Starting scala interpreter...
 [info]
-Welcome to Scala 2.13.10 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_112).
+Welcome to Scala 3.3.6 (21.0.2, Java OpenJDK 64-Bit Server VM)
```

Comment:
I updated this message just for consistency.
It also shows **Java** `21` instead of `8` to show it supports newer technologies, maybe we should use `17` instead since that will be the new minimum?